TESTS/host_tests/pyusb_basic.py

```diff
@@ -122,7 +309,7 @@ def test_device(serial_number, log=print):
         ## -- other tests-- ##
         #-report throughput for in/out of control, bulk, interrupt and iso transfers
         #-verify that construction/destruction repeatedly works gracefully
-
+        return True
```

Comment:
Why does this return in the middle of the function?

Reply:
This function is used only for documentation purpose and will be removed soon